pkg/secrets/check_rights_windows.go

```diff
@@ -24,7 +24,7 @@ var (
 
 // checkRights check that the given filename has access controls set only for
 // Administrator, Local System and the datadog user.
-func checkRights(filename string) error {
+func checkRights(filename string, _ bool) error {
```

Comment:
This option will be mainly used in `cluster-agent` and we release the cluster-agent for linux.
Since the agent still need to run as root, this option is not very useful.

Reply:
I have added a comment in the file.